### DISCUSSION TITLE
HidHide Updater interrupts game

### DISCUSSION THREAD

**Discussion by avi819** [nefarius/HidHide#143] [General]
I started getting issues with my computer going dark and then to lock screen while actively gaming (Elden Ring) about two weeks ago. First time I've ever been interrupted while gaming was when HidHide's updater yanks me from the game back to desktop to show me that a new update is available. I then turned off automatic update for HidHide and updated to HidHide_1.4.202_x64.

Since then, my computer started going to lock screen randomly about once or twice a day while actively gaming. Is this a HidHide issue? I have game mode turned on, and has never encountered any game interruption until HidHide's updater interrupted it.

 I updated to HidHide_1.5.212_x64 a day ago. Still getting lock screen once or twice a day while gaming.

**Comment by nefarius** [comment]
Nothing in any HidHide code triggers a lock screen.

**Reply by avi819** [reply]
Is it possible that it makes Windows think there is no input from USB devices, so it then turns off screen / go to lock screen? Maybe unrelated, but I noticed also my monitor seems to want to turn off during active use.

Today I set "When plugged in, turn off my screen after" to Never. It hasn't gone to lock screen so far.

**Reply by nefarius** [reply]
No, HidHide doesn't intercept system services that communicate with HID devices, the reported behaviour is entirely unrelated to HidHide.

**Reply by nefarius** [reply]
I can't really think of anything that would apply to your case. However, please try to disable the HidHide updater scheduled task temporarily and observe if anything changes.

**Reply by avi819** [reply]
Thank you for the response. I found the updater’s manual update setting once before, but can’t anymore. Can you point me to where I can find and disable the HidHide updater scheduled tasks?

Thank you so much.

**Reply by nefarius** [reply]
Same location since Windows Vista, where all the other scheduled tasks live 😛 

![image](https://github.com/nefarius/HidHide/assets/286631/9dabc6e6-71f3-4e99-8f19-6be83cb14afa)

**Comment by avi819** [comment]
Thank you! Found two and disabled both.

![Untitled-2](https://github.com/nefarius/HidHide/assets/55117604/4c5301ec-81b1-47d5-8ea0-3a8191e2afde)

**Reply by nefarius** [reply]
The top one is legacy now and should be gone after you installed v1.5 so something funky going on with your system anyway 👀
